paket.dependencies

```diff
@@ -1,29 +1,62 @@
-source https://nuget.org/api/v2
-framework: netstandard2.0, netcoreapp2.0
-redirects: force
-
-nuget FSharp.Core >= 4.2.3
-nuget TypeShape >= 8.0.1
-nuget Microsoft.Extensions.Configuration
+source https://www.nuget.org/api/v2
+source https://api.nuget.org/v3/index.json
+storage: none
+nuget FSharp.Core 5.0
```

Comment:
@tamizhvendan There is one thing I was not sure about. Which versions of dependencies you want here? The new ones are pretty restrictive. The library still targets `netstandard2.0` though.

Reply:
It looks good to me and I feel we can retain it.